File: common/src/main/java/com/cainsgl/common/dto/response/LoginResponse.java
package com.cainsgl.common.dto.response;

import com.cainsgl.common.entity.user.UserEntity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponse {
    
    /**
     * 登录token
     */
    private String token;
    
    /**
     * 用户信息
     */
    private UserEntity userInfo;
}

File: user/src/main/java/com/cainsgl/user/dto/response/LoginResponse.kt
package com.cainsgl.user.dto.response

import com.cainsgl.common.entity.user.UserEntity

data class LoginResponse(
    val token: String,
    val userInfo: UserEntity?,
    val isNew:Boolean=false
)
data class ErrorLoginResponse(
    val msg: String,
    val needCaptcha: Boolean = false,
    val captchaImage: String? = null, // Base64 编码的验证码图片
    var modal:String?=null,
    var token:String?=null
)
data class RegisterSuccessResponse(
    val token: String,
)